ush/global_ens/evs_global_ens_atmos_grid2grid.sh

```diff
@@ -484,16 +549,25 @@ if [ $verify = precip ] ; then
         export err=$?; err_chk
       fi
     fi
-  done #end of metplus_jobs
+  done # end of metplus_jobs loop
+
+  # Indicate all tasks are completed
+  >$COMOUTsmall_precip/stats_completed
+  echo "stats are completed" >> $COMOUTsmall_precip/stats_completed
 
-  #******************************************************
-  # Collect small stat files to form a final big stst file
-  #*******************************************************
+  # Clean up restart directory
+  rm -rf $COMOUTsmall_precip/restart/grid2grid/run_${modnam}_ccpa*
```

Comment:
I don't think we can be removing directories from COMOUTsmall like this.

@AliciaBentley-NOAA @AndrewBenjamin-NOAA @PerryShafran-NOAA Thoughts?

Reply:
Same as above.